### ISSUE TITLE
[Feature Request] Amazon Alexa Support

### ISSUE THREAD

**Issue by louisefindlay23** [jellyfin/jellyfin#917]
Not a high priority but will Emby's Premier only Alexa skill be ported to Jellyfin? https://emby.media/announcing-the-emby-alexa-skill.html

**Comment by JustAMan** [comment]
If that is premiere only we probably don't have a source to port.
So if so wants to they will have to implement it from scratch. Not saying it's impossible, though.

**Comment by anthonylavado** [comment]
Having researched a lot into Alexa for another project I help maintain, this would definitely be a custom effort.

Depending on how it's done, it may cost money too - not big $$, but some money for AWS. Will have to discuss later.

**Comment by louisefindlay23** [comment]
@JustAMan @anthonylavado It may work through Jellyfin for Kodi as there is a [Kodi Alexa skill ](https://forum.kodi.tv/showthread.php?tid=327562)(accessible through beta testing invites) that I've used in the past. Currently using the Plex one at the moment in PlexKodiConnect. 

Might need to use direct paths but I plan on migrating one of my clients to Jellyfin for Kodi next week so I can try it out. I already run the server in docker from when I did dvr testing.
